cmd/tk/status.go

```diff
@@ -32,15 +34,24 @@ func statusCmd() *cli.Command {
 		context := status.Client.Kubeconfig.Context
 		fmt.Println("Context:", context.Name)
 		fmt.Println("Cluster:", context.Context.Cluster)
+
+		fmt.Println()
+		fmt.Println("Directory Layout:")
+		fmt.Println("  Project root:", status.DirLayout.Root)
+		fmt.Println("  Environment:", status.DirLayout.Base)
+		fmt.Println("  Entrypoint:", status.DirLayout.Entrypoint)
+
+		fmt.Println()
 		fmt.Println("Environment:")
-		for k, v := range structs.Map(status.Env.Spec) {
-			fmt.Printf("  %s: %v\n", k, v)
+		for _, f := range structs.Fields(status.Env.Spec) {
+			fmt.Printf("  %s: %v\n", f.Name(), mustJsonFmt(f.Value()))
```

Comment:
While playing with `tk status` and `fzf`, I noticed the order of these fields change. Can we ensure these are sorted for consistency?